sdk/python/foundation-models/system/content-safety/deploy_with_content_safety.ipynb

```diff
@@ -0,0 +1,803 @@
+    "print(endpoint_name)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 2. Connect to your AML Workspace"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "ExecuteTime": {
+     "end_time": "2023-07-04T09:28:38.486754700Z",
+     "start_time": "2023-07-04T09:28:35.703401800Z"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "import os, json\n",
+    "from azure.ai.ml import MLClient, Input, Output \n",
+    "from azure.identity import DefaultAzureCredential, InteractiveBrowserCredential\n",
+    "\n",
+    "try:\n",
+    "    credential = DefaultAzureCredential()\n",
+    "    # Check if given credential can get token successfully.\n",
+    "    credential.get_token(\"https://management.azure.com/.default\")\n",
+    "except Exception as ex:\n",
+    "    # Fall back to InteractiveBrowserCredential in case DefaultAzureCredential not work\n",
+    "    credential = InteractiveBrowserCredential()\n",
+    "\n",
+    "try:\n",
+    "    ml_client = MLClient.from_config(credential=credential)\n",
+    "except Exception as ex:\n",
+    "    # NOTE: Update following workspace information to contain\n",
+    "    #       your subscription ID, resource group name, and workspace name\n",
+    "    client_config = {\n",
+    "        \"subscription_id\": subscription_id,\n",
+    "        \"resource_group\": resource_group,\n",
+    "        \"workspace_name\": workspace_name,\n",
+    "    }\n",
+    "    # write and reload from config file\n",
+    "    config_path = \"../.azureml/config.json\"\n",
+    "    os.makedirs(os.path.dirname(config_path), exist_ok=True)\n",
+    "    with open(config_path, \"w\") as fo:\n",
+    "        fo.write(json.dumps(client_config))\n",
+    "    ml_client = MLClient.from_config(credential=credential, path=config_path)\n",
+    "    \n",
+    "print(ml_client)\n",
+    "workspace_location = ml_client.workspaces.get(ml_client.workspace_name).location\n",
+    "subscription_id = ml_client.subscription_id\n",
+    "resource_group_name = ml_client.resource_group_name\n",
+    "workspace_name = ml_client.workspace_name\n",
+    "print(workspace_location)\n",
+    "print(subscription_id)\n",
+    "print(resource_group_name)\n",
+    "print(workspace_name)\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 4. Create Azure AI Content Safety"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### 4.1 Choose a region for your Content Safety\n",
+    "Currently, Azure AI Content Safety only available in the following regions:\n",
+    "- East US\n",
+    "- West Europe\n",
+    "- Central US EUAP"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "aacs_location = \"east us\" # please choose the nearest location to your workspace\n",
```

Comment:
Can we add something like this? So user doesn't need to guess the casing / format of the region? (Does it only work for all lower case? Or must it be some other format, for example)
```
available_acs_locations = ['east us', 'west europe']
```

Reply:
Added.